src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT04-23-expected.json

```diff
@@ -234,15 +234,21 @@
         },
         "status": "unknown",
         "location": [
+          {
+            "status": "active"
+          },
+          {
+            "status": "completed"
+          },
```

Comment:
How about these 2 status? They look redundant.

Reply:
These two status are comming from different sources, and used to represent different Location status. In the test data we don't have adequate Location related segments, so there will be a single status here.